apps/explorer/src/ui/Button.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva, type VariantProps } from 'class-variance-authority';
+
+import ButtonOrLink, { type ButtonOrLinkProps } from './utils/ButtonOrLink';
+
+const buttonStyles = cva(
+    [
+        'inline-flex items-center justify-center',
+        // TODO: Remove when CSS reset is applied.
+        'cursor-pointer font-sans no-underline',
+    ],
+    {
+        variants: {
+            variant: {
+                primary:
+                    'bg-sui-dark text-sui-light hover:text-white border-none',
```

Comment:
I understand this CSS in JS has more to do with `class-variance-authority` but I am not a fan.